### DISCUSSION TITLE
how to find atom that is furthest away from other atoms..

### DISCUSSION THREAD

**Discussion by orgw** [rdkit/rdkit#6010] [Q&A]
Hi i'm trying to find why a certain ligand structure has an atom that's apart from other atoms such as the following:

![image](https://user-images.githubusercontent.com/64889801/214767059-3e24c761-40b2-423b-8266-6b1c8f45578c.png)

this is an atom from the CASP dataset. 

I want to remove this atom. What steps should i take?

**Comment by greglandrum** [comment]
@orgw that looks like the kind of thing that happens when one or more atoms have all coordinates = 0. If that's true, then you just want to look for those atoms.
You can quickly check this by doing: `print(Chem.MolToV3KMolBlock(mol))` where `mol` is the molecule you have constructed.
